spec/services/service_limits_spec.rb

```diff
@@ -209,6 +209,37 @@
       end
     end
 
+    context 'whitespace annotations' do
+      let!(:user) { create(:new_user, num_annotation_characters: user_annotation.length ) }
+      let!(:highlight) { create(:highlight, user: user, annotation: user_annotation) }
+
+      let(:user_annotation) { 'annotation' }
+
+      it 'will update when annotation is set to empty string' do
+        expect do
+          service_limits.with_update_protection do
+            highlight.annotation = ""
```

Comment:
very minor point (optional). i would stay consistent w/ single (or double) quotes for non-interpreted strings if possible.

Reply:
I've left `" \n"` since api seems to count `\n` as one character and single quotes would interpret it literally.